geoh5py/ui_json/enforcers.py

```diff
@@ -259,6 +263,34 @@ def validations(self, val: Any):
         self._validations = val
 
 
+class TypeUIDEnforcer(Enforcer):
+    """
+    Enforces restricted type uid(s).
+
+    :param validations: Valid geoh5py object type uid(s).
+
+    :raises TypeValidationError: If value is not a valid type uid.
+    """
+
+    enforcer_type = "type_uid"
+
+    def __init__(self, validations: list[str]):

```

Comment:
I just opened up a new issue to implement the RequiredWorkspaceObjectsEnforcer and ran into this while writing the a test.  It will be addressed in GEOPY-1093